### PR TITLE
Content negotiation, additional examples

### PR DESCRIPTION
**Pull request by dgarijo**
Support for:

- Content negotiation (you can add uris like https://purl.org/net/p-plan)
- Loading ontologies from local files (useful for drafting)
- Added examples documenting both cases

### CODE REVIEW

File: src/test/java/edu/isi/oba/ObaUtilsTest.java [hunk 1]
```diff
@@ -1,5 +1,7 @@
 package edu.isi.oba;
 
+import com.github.jknack.handlebars.internal.Files;
```
Comment (sirspock) [inline src/test/java/edu/isi/oba/ObaUtilsTest.java:3]: Do we need this?

File: src/main/java/edu/isi/oba/ObaUtils.java [hunk 2]
```diff
@@ -271,5 +283,54 @@ public static JSONObject read_json_file(String file_name) throws IOException {
...
     }
+    
+    /**
+    * Method that will download an ontology given its URI, doing content
+    * negotiation The ontology will be downloaded in the first serialization
+    * available (see Constants.POSSIBLE_VOCAB_SERIALIZATIONS)
+    * @param uri the URI of the ontology
+    * @param downloadPath path where the ontology will be saved locally.
+    */
+    public static void downloadOntology(String uri, String downloadPath) {
```
Comment (sirspock) [inline src/main/java/edu/isi/oba/ObaUtils.java:294]: What is the benefits of context negotiation?

File: src/test/java/edu/isi/oba/ObaUtilsTest.java (hunk 1 above)
Comment (dgarijo) [reply src/test/java/edu/isi/oba/ObaUtilsTest.java:3]: no, will remove

Comment (dgarijo) [reply src/test/java/edu/isi/oba/ObaUtilsTest.java]: Fixed in https://github.com/KnowledgeCaptureAndDiscovery/OBA/pull/96/commits/d694104a837a295404aae79fb36b6aee43cd0524

**Pull request merged**